### PR TITLE
add options to fetch triples/graph from an external KG

### LINKED ISSUE #11: Update mapping strategy to fetch a named graph from a triplestore and append this to the DLite collection.

### PR DESCRIPTION
**Pull request by Treesarj**
# Description:

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand, including clearly named variables?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

File: oteapi_dlite/strategies/oceanlab_influx_parser.py [hunk 1]
```diff
@@ -89,9 +89,7 @@ class InfluxParseParseConfig(AttrDict):
 
     measurements: Annotated[
         list[dict],
-        Field(
-            description="list of measurements containing measurement and field values as list of dictionary"
-        ),
+        Field(description="Measurement and field values as list of dictionary"),
```
Comment (CasperWA) [inline oteapi_dlite/strategies/oceanlab_influx_parser.py]: I'm guessing this is something left over from the other commits/PR?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cachetools>=5.3.3
 DLite-Python>=0.4.5,<1.0
-fastapi>=0.111.0
+fastapi<=0.110.0
```
Comment (CasperWA) [inline requirements.txt]: Really?

**Review by CasperWA — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (Treesarj) [reply requirements.txt]: autoimport , removed ! now using python logging

File: oteapi_dlite/strategies/oceanlab_influx_parser.py (hunk 1 above)
Comment (Treesarj) [reply oteapi_dlite/strategies/oceanlab_influx_parser.py]: yes

**Pull request merged**